ui/src/app/pages/workspace/create-billing-account-modal.spec.tsx

```diff
@@ -62,7 +62,7 @@ describe('CreateBillingAccountModal', () => {
       'tester@mactesterson.edu><script>alert("hello");</script>'
     );
     expect(getHTMLInputElementValue(getByTestId('user-workbench-id'))).toEqual(
-      'tester@fake-research-aou.org'
```

Comment:
After searching for the standard user for running tests, I found a few instances of the string.  Replace these with the const